lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -3334,6 +3334,55 @@ void mme_app_handle_handover_required(
   OAILOG_FUNC_OUT(LOG_MME_APP);
 }
 
+void mme_app_handle_handover_request_ack(
+    mme_app_desc_t* mme_app_desc_p,
+    itti_s1ap_handover_request_ack_t* const handover_request_ack_p) {
+  OAILOG_FUNC_IN(LOG_MME_APP);
+
+  struct ue_mm_context_s* ue_context_p          = NULL;
+  MessageDef* message_p                         = NULL;
+  itti_mme_app_handover_command_t* ho_command_p = NULL;
+
+  OAILOG_INFO(
+      LOG_MME_APP,
+      "Received handover request ack from S1AP for ue-id " MME_UE_S1AP_ID_FMT
+      "\n",
+      handover_request_ack_p->mme_ue_s1ap_id);
+
+  ue_context_p = mme_ue_context_exists_mme_ue_s1ap_id(
+      handover_request_ack_p->mme_ue_s1ap_id);
+
+  if (!ue_context_p) {
+    OAILOG_ERROR(
+        LOG_MME_APP, "UE context doesn't exist for UE " MME_UE_S1AP_ID_FMT "\n",
+        handover_request_ack_p->mme_ue_s1ap_id);
+  }
+
+  message_p    = itti_alloc_new_message(TASK_MME_APP, MME_APP_HANDOVER_COMMAND);
```

Comment:
handle failure in allocation

Reply:
Good catch, also added an early return if we get an invalid `mme_ue_s1_id` above.